cpp/ql/lib/upgrades/initial/semmlecode.cpp.dbscheme

```diff
@@ -777,7 +777,7 @@ case @expr.kind of
 | 106 = @hastrivialconstr // __has_trivial_constructor ::= type
 | 107 = @hastrivialcopy // __has_trivial_copy ::= type
 | 108 = @hasuserdestr // __has_user_destructor ::= type
-| 109 = @hasvirtualdestr // __has_virtual_desctructor ::= type
+| 109 = @hasvirtualdestr // __has_virtual_destructor ::= type
```

Comment:
?

Reply:
To add we should fix this if it still occurs in the current `dbscheme` file, but not in this particular one, as this one is part of the upgrade/downgrade mechanism we have for these files.